guava-tests/benchmark/com/google/common/collect/IteratorBenchmark.java

```diff
@@ -32,7 +33,7 @@
 
   // use concrete classes to remove any possible polymorphic overhead?
   Object[] array;
-  ArrayList<Object> arrayList;
+  List<Object> arrayList;
   LinkedList<Object> linkedList;
```

Comment:
How about this `LinkedList` as well?

Reply:
How about this "use concrete classes to remove any possible polymorphic overhead?" comment? ;)

Reply:
Ah, I missed that. Silly me. :P

Reply:
I shall refactor to remote all concrete classes. Thanks @perceptron8

Reply:
@fvarg00 Uh, I _think_ what the comment is saying is that concrete classes are purposely used _instead_ of the `List` interface to "remove any possible polymorphic overhead" (not entirely sure what that means), so AFAICT this PR would probably not be accepted even if the `LinkedList` is refactored...

Reply:
@jbduncan - thanks for your comment. It makes sense now.